src/LeetCode/Java_Problems/LeetCode_36.java

```diff
@@ -0,0 +1,47 @@
+package LeetCode.Java_Problems;
+
+//LINK: https://leetcode.com/problems/valid-sudoku/
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class LeetCode_36 {
+	public boolean isValidSudoku(char[][] board) {
+		int len = board.length;
+		Set<Character>[] rows = generateArrays(len);
+		Set<Character>[] columns = generateArrays(len);
+		Set<Character>[] subBoxes = generateArrays(len);
+
+		for (int i = 0; i < len; i++) {
+			for (int j = 0; j < len; j++) {
+				char curr = board[i][j];
+				int subBox = i / 3 * 3 + j / 3;
+				if (curr == '.') {
+					continue;
+				}
+				if (checkCurrentChar(rows[i], columns[j], subBoxes[subBox], curr)) {
+					return false;
+				} else {
```

Comment:
Dlaczego else jesli zadziala return?

Reply:
jesli metoda zwroci true to znaczy ze liczba sie powtarza w ktorejs kolumnie/rzedzie/subbox i sudoku nie jest valid, ale jesli zwroci false to musze ta liczbe dodac do setow, aby potem przejsc do kolejnej liczby